edxval/api.py

```diff
@@ -182,6 +143,52 @@ def update_video_status(edx_video_id, status):
     video.save()
 
 
+def get_course_video_image_url(course_id=None, edx_video_id=None, video_image=None):
+    """
+    Returns course video image url.
+
+    Arguments:
+        course_id: ID of course
+        edx_video_id: ID of the video
+        video_image: VideoImage instance
+
+    Returns:
+        course video image url or None if no image found
+    """
+    storage = get_video_image_storage()
+
+    try:
+        if video_image is None:
+            video_image = CourseVideo.objects.get(course_id=course_id, video__edx_video_id=edx_video_id).video_image
+    except ObjectDoesNotExist:
+        return None
+
+    return storage.url(video_image.image.name)
+
+
+def update_video_image(edx_video_id, course_id, image_data, file_name):
+    """
+    Update video image for an existing video.
+
+    Arguments:
+        edx_video_id: ID of the video.
+        course_id: ID of course.
+        image_data: Image data for video.
+        file_name: File name of the image file.
```

Comment:
I think the argument description is too obvious and not providing any extra context here. Better to remove them ? We can leave image_data as it by adding the type (InMemoryUploadedFile).